### PR TITLE
Convert Spells and Runes to RevScript

### PR DESCRIPTION
**Pull request by beats-dh**
**All player spells and runes have been converted, the monsters spells are still in progress! All need revision.**

**With the contribution of @LeoTK4**

### **_Spells:_**

- [x] Attack
- [x] Conjuring
- [x] Healing
- [x] House
- [ ] Monster A --->> F (z)
- [x] Party
- [x] Runes
- [x] Summon
- [x] Support

### CODE REVIEW

**Comment by adrianomengue** [comment]
Wow!

File: data/scripts/spells/runes/paralyze rune.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
+local combat = Combat()
+combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_MAGIC_RED)
+
```
Comment (adrianomengue) [inline data/scripts/spells/runes/paralyze rune.lua]: Can you take a look into my PR? So we don't have any conflict when merge it. #2511

**Comment by adrianomengue** [comment]
As we can see here #1845, in bomb/field rune's scripts we should add this attribute **setPzLocked="1"**

File: data/scripts/spells/runes/paralyze rune.lua (hunk 1 above)
Comment (beats-dh) [reply data/scripts/spells/runes/paralyze rune.lua]: Done.

**Comment by Beats-Dh** [comment]
> Como podemos ver aqui [# 1845](https://github.com/opentibiabr/otservbr-global/pull/1845) , nos scripts de runas de bomba / campo devemos adicionar este atributo **setPzLocked = "1"**

**Comment by beats-dh** [comment]
> As we can see here #1845, in bomb/field rune's scripts we should add this attribute **setPzLocked="1"**

Done.

**Pull request closed without merging**